ngraph/frontend/onnx_import/include/onnx_import/core/model.hpp

```diff
@@ -27,6 +27,10 @@ namespace ngraph
 {
     namespace onnx_import
     {
+        /// \brief      Type of container which stores opset version and domain in ONNX format
+        using OpsetImportType =
```

Comment:
Just for consideration: since this type aliases a repeated field, meaning it can contain more than one object, would it make sense to call this alias something like `OpsetImports`? Accordingly you could rename the getter `get_opset_imports` in graph.hpp

Reply:
@tomdol right, it sounds much better, done